style_admin.css

```diff
@@ -28,31 +28,43 @@ body {
 	z-index: 1;
 }
 
-/* Style for content_1, which contains the grid system for the login box and header */
-.content_1 {
-	background: lightblue;
-	opacity: ;
-	font-size: 1.1vw;
-	margin: 0vw 0vw 0vw 0vw;
-	grid-column: 1 / span 2;
-	grid-row: 2;
-	display: grid;
-	grid-template-columns: 36vw 27.5vw 36vw;
-	grid-template-rows: auto;
-	z-index: 1;
-}
-
 /* Style for login box */
 .login_box {
 	background: #ccb0f7;
 	box-sizing: border-box;
-	border-radius: 5%;
-	position: relative;
-	grid-column: 2;
-	grid-row: 1;
+	grid-column: 1;
+	grid-row: 2;
 	display: grid;
 }
 
+/* Style for text input */
+input[type=text] {

```

Comment:
Below is where I added style for the text input field,  password input field and submit input button.